gui_qt/check_report.py

```
@@ -288,7 +288,7 @@ def summarize_manifest_writeback(manifest: dict[str, object]) -> WritebackSummar
         return WritebackSummary(
             status="warn",
             heading="需要先处理问题",
-            message="最近一次检查结果为需处理，不应写回。",
+            message="最近一次检查结果为需处理，不应写回。可点击「补救命令」查看 retry 参考命令，处理后重新检查；只有 safe 才能写回。",
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)